docs/manual/advanced-features.tex

```diff
@@ -1247,7 +1250,7 @@
 % The Checker Framework is best at reasoning about Java expressions that
 % are variable references, but these expressions are not.
 \begin{itemize}
-\item Class literals with primitive type (int.class) or with array type (String[].class).
+\item Class literals with with array type (String[].class).
```

Comment:
This pull request added support for `String[].class` right?

Reply:
It does now.